<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeleccionarEstiloDlg</class>
 <widget class="QDialog" name="SeleccionarEstiloDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selección de Estilo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/aplicacion/images/alephLogoIcon.ico</normaloff>:/aplicacion/images/alephLogoIcon.ico</iconset>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Seleccione el estilo que desea para su aplicación</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkDarkVersion">
     <property name="text">
      <string>Usar versión en colores oscuros</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bbBotones">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
